bindings/wasm/src/error.rs

```diff
@@ -130,3 +132,52 @@ impl From<UpdateError> for WasmError<'_> {
     }
   }
 }
+
+/// Convenience struct to convert Result<JsValue, JsValue> to an AccountResult<_, AccountError>
+pub struct JsValueResult(pub(crate) Result<JsValue>);
```

Comment:
See the previous comment on possibly refactoring `Storage` errors, it might make `JsValueResult` unnecessary since errors would be mapped to e.g. `StorageError` variants internally?